tools/zulip-icon-debug.svg

```diff
@@ -0,0 +1,7 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 24 24">
+<image xlink:href="../../zulip/static/images/logo/zulip-icon-circle.svg" width="24" height="24"/>
+<g>
+<rect x="0" y="19" rx="3" ry="5" width="24" height="5" fill="#ff5050"/>
```

Comment:
This radius is still more than half the height. You’re rounding with ellipses, not circles.

Reply:
I'm sorry @andersk! Made the necessary changes now. Please review it!